docs/SchemaMigrationGuide.md

```diff
@@ -1,98 +1,89 @@
 # Schema Migration Guide
-This guide describes how to upgrade the schema for the FHIR server. The upgrade scripts are the T-SQL script that alters the database in some way and has a version encoded in its filename. The version will be an incrementing integer.
+This document describes how to perform Sql schema migration on the Open source FHIR server.
```

Comment:
In the readme we use FHIR Server for Azure to refer to the project. Should we use that here too?